core/src/test/java/kafka/server/share/SharePartitionManagerTest.java

```diff
@@ -127,13 +128,14 @@
 @SuppressWarnings({"ClassDataAbstractionCoupling"})
 public class SharePartitionManagerTest {
 
-    private static final int RECORD_LOCK_DURATION_MS = 30000;
+    private static final int DEFAULT_RECORD_LOCK_DURATION_MS = 30000;
     private static final int MAX_DELIVERY_COUNT = 5;
     private static final short MAX_IN_FLIGHT_MESSAGES = 200;
     static final int PARTITION_MAX_BYTES = 40000;
     private static final int DELAYED_SHARE_FETCH_MAX_WAIT_MS = 2000;
     private static final int DELAYED_SHARE_FETCH_PURGATORY_PURGE_INTERVAL = 1000;
     private static final int DELAYED_SHARE_FETCH_TIMEOUT_MS = 3000;
+    private static final GroupConfigManager GROUP_CONFIG_MANAGER = new GroupConfigManager(new HashMap<>());
```

Comment:
I think it's a bit unusual using capitalisation for the name of a static instance of an object rather than a literal.

Reply:
Thanks for the review. If I change the naming to camelcase, the checkkstyle rules won't allow to build the project, that's why I went ahead with this. The other thing I could do is to not have a static variable for groupConfigManager at all and simply pass a new object in the constructor for SharePartitionManager. That seems to be the better thing, so I will do that for now. Let me know if any further changes are required, thanks !